wallet/src/de/schildbach/wallet/ui/EditProfileActivity.kt

```diff
@@ -104,6 +113,41 @@ class EditProfileActivity : BaseMenuActivity() {
             }
 
         })
+
+        editProfileViewModel.updateProfileRequestState.observe(this, Observer {
+            if (it != null) {
+                when (it.status) {
+                    Status.SUCCESS -> {
+                        Toast.makeText(this@EditProfileActivity, "Update successful", Toast.LENGTH_LONG).show()
+                        //setEditingState(false) // if this line is here, then the Profile changes to the previous value (crazy?)
```

Comment:
This is where I thought we could turn the observer back ON, but it the data loaded was old.